src/psyclone/psyGen.py

```diff
@@ -4990,12 +4990,25 @@ def _if_stmt_handler(self, node, parent):
                            nodes_parent=node)
         return ifblock
 
+    def _return_handler(self, node, parent):
```

Comment:
pylint complains that `node` is unused and also that this could be a static function.

Reply:
I knew about this one (`ignore_handler` had the same problem) but I am unsure what is the best fix. The problem here is that the handler(node, parent) is an interface equal for all handlers and it will be called with this two arguments by resolving dynamically which handler it calls.

One solution is to use the `_` as first argument, but then I am not sure how to write the docstring (is `:param _ :` ok?). The second is to add the `pylint: disable=unused-argument` (currently in `ignore_handler`)

The static function message is even more difficult to solve, it also happens just in a few handlers because most of them do use self, but I don't think I can convert just a few of them to static functions because the call would be different (and it just looks odd). Should I maybe add a pylint disable comment here as well?

Reply:
Ah, yes, I wondered whether this was conforming to some uniform interface. I think we should use `_` for the argument and then not document them at all in the docstring, apart from a comment to say that they are there for compatibility with XXX interface.
As for the static function, I think a pylint disable is the way to go. BTW, I don't think the call would be any different because you can still just do `some_obj.some_static_method()`.

Reply:
That works for the `return_handler(_, parent)`, but the `ignore_handler(_, _)` which also is not using parent complains with:
```
 SyntaxError: duplicate argument '_' in function definition
```
Pylint accepts ` def _ignore_handler(self, _1, _2):` is this readable enough? I will add the docstring note anyway.

Reply:
`def _ignore_handler(self, *_):`  is also accepted by pylint for a list of ignored arguments

Reply:
That combined with a docstring note seems like a nice solution.